bal_tools/pools_gauges.py

```diff
@@ -116,31 +116,39 @@ def query_root_gauges(self, skip=0, step_size=100) -> list:
             result += self.query_root_gauges(skip + step_size, step_size)
         return result
 
-    def query_all_gauges(self, include_other_gauges=True) -> list:
+    def query_all_gauges(self, include_other_gauges=True) -> List[GaugeData]:
         """
         query all gauges from the apiv3 subgraph
         """
         data = self.subgraph.fetch_graphql_data("apiv3", "get_gauges", {"chain": self.chain.upper()})
         all_gauges = []
-        for gauge in data["poolGetPools"]:
-            if gauge['staking'] is not None:
-                if gauge['staking']['gauge'] is not None:  # this is an edge case for the 80bal20weth pool
-                    all_gauges.append({"id": gauge['staking']['gauge']['gaugeAddress'], "symbol": f"{gauge['symbol']}-gauge"})
-                    if include_other_gauges:
-                        for other_gauge in gauge['staking']['gauge']['otherGauges']:
-                            all_gauges.append({"id": other_gauge['id'], "symbol": f"{gauge['symbol']}-gauge"})
+        for pool in data["poolGetPools"]:
+            gauge_pool = GaugePoolData(**flatten_nested_dict(pool))
+            if gauge_pool.staking is not None and gauge_pool.staking.get('gauge') is not None:
+                gauge = gauge_pool.staking['gauge']
+                all_gauges.append(GaugeData(
+                    id=gauge['gaugeAddress'],
+                    symbol=f"{gauge_pool.symbol}-gauge"
+                ))
+                if include_other_gauges:
+                    for other_gauge in gauge.get('otherGauges', []):
+                        all_gauges.append(GaugeData(
+                            id=other_gauge['id'],
+                            symbol=f"{gauge_pool.symbol}-gauge"
+                        ))
         return all_gauges
 
-    def query_all_pools(self) -> list:
+    def query_all_pools(self) -> List[PoolData]:
         """
         query all pools from the apiv3 subgraph
         filters out disabled pools
         """
         data = self.subgraph.fetch_graphql_data("apiv3", "get_pools", {"chain": self.chain.upper()})
         all_pools = []
         for pool in data["poolGetPools"]:
-            if pool['dynamicData']['swapEnabled']:
-                all_pools.append({"address": pool['id'], "symbol": pool['symbol']})
```

Comment:
note the `pool['id']` data here is now referenced with `id` rather than `address`

Reply:
Think PoolID and Address are different.  Suggest we not remove whatever was already there from results, but it could make sense to add the other

Reply:
good catch; fixed in 90061d008afcaf1d8bfa6d50a433f9941e4250ff